cis/publisher.py

```diff
@@ -118,6 +124,42 @@ def _validate_profile_data(self):
         # Validate data prior to sending to CIS
         return validation.Operation(self.publisher, self.profile_data).is_valid()
 
+    def _reintegrate_profile_with_api(self):
+        person = api.Person(
+            person_api_config={
+                'audience': self.config('person_api_audience', namespace='cis'),
+                'client_id': self.config('oidc_client_id', namespace='cis'),
+                'client_secret': self.config('oidc_client_secret', namespace='cis'),
+                'oidc_domain': self.config('oidc_domain', namespace='cis'),
```

Comment:
nit: technically its just oauth2 not oidc

Reply:
fixed